### ISSUE TITLE
Add Search Translator for PMC

### ISSUE THREAD

**Issue by adam3smith** [zotero/translators#1782]
Suggested here: https://github.com/zotero/translation-server/issues/51#issuecomment-438552462

This makes sense to me and since PMC IDs, are quite distinctive (`PMC\d+`), there wouldn't appear to be any downside. Not that this would require a small patch on the Zotero end, too.

**Comment by dhimmel** [comment]
Makes sense to me.

Possibly the new NCBI [Literature Citation Exporter](https://api.ncbi.nlm.nih.gov/lit/ctxp) could be useful (although it sounds like you already support metadata retrieval).

One question is how would you deal with records that have multiple versions? As an example, `PMC5676196` has 5 versions. For example, see version 4 at https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5676196.4/. Ideally, it would be nice to cite a specific version, although I'm not sure of the technical feasibility or desirability of this.

**Comment by adam3smith** [comment]
yeah, I think we'll just use eutils because we have this all set up already. As for versions -- I think the only question is if we need to include the version number somehow in the metadata. While CSL does have a version variable, Zotero doesn't really apply this to journal articles, nor do I think it currently makes a lot of sense, so I'd just import the version specified in the PMC ID, i.e. version 4 for PMC5676196.4 and the most recent version (5 in this case) for PMC5676196.

**Comment by dhimmel** [comment]
> I'd just import the version specified in the PMC ID, i.e. version 4 for PMC5676196.4 and the most recent version (5 in this case) for PMC5676196.

I agree this is the most sensible behavior. If no version specified, use latest. Otherwise use the specified version.

> Zotero doesn't really apply this to journal articles, nor do I think it currently makes a lot of sense,

As publishing becomes more iterative, version numbers may start to matter more. For F1000 which let's authors completely edit their manuscripts, the versioning is important. However, I still don't think it's very common.
